Sources/TokamakCore/Fiber/Layout/TextLayoutComputer.swift

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 5/24/22.
+//
+
+import Foundation
+
+/// Measures the bounds of the `Text` with modifiers and wrapping inside the `proposedSize`.
```

Comment:
nit: if I understand the purpose of this class correctly. BTW: does `LayoutComputer` have to be a `class`? Could we use `structs` for these `LayoutComputer` types to avoid dynamic allocations?

```suggestion
/// Measures the bounds of the `Text` with modifiers and wraps it inside the `proposedSize`.
```